bin/esparse.js

```diff
@@ -90,22 +97,14 @@ process.argv.splice(2).forEach(function (entry) {
         options.tokens = true;
     } else if (entry === '--tolerant') {
         options.tolerant = true;
-    } else if (entry.slice(0, 2) === '--') {
+    } else if (entry === '--') {
+        forceFile = true;
```

Comment:
This will cause mishandling of `bin/esparse.js - --`, but that seems rare enough that I didn't consider it worth adding extra code to address the interactions.